example/cumulus-tf/variables.tf

```diff
@@ -49,6 +49,27 @@ variable "launchpad_passphrase" {
   type    = string
   default = ""
 }
+variable "lzards_launchpad_certificate" {
+  type    = string
+  default = "launchpad.pfx"
+}
+
+variable "lzards_launchpad_passphrase" {
+  type    = string
+  default = ""
+}
+
+variable "lzards_api" {
+  description = "LZARDS API endpoint"
+  type        = string
+  default     = "https://lzards.sit.earthdata.nasa.gov/api/backups"
```

Comment:
Is this the API that we can hit from our sandbox?

Reply:
Correct, they opened rules for their SIT env to our SIT/sandbox.